src/utils/polyfills.ts

```
@@ -1,33 +1,15 @@
-
+    // Add minimal, safe polyfills if needed
+    // Example: window.fetch is available in modern browsers; add guards as needed
+    // Currently no polyfills required; keep as no-op to avoid blocking startup
   } catch (error) {
-    console.warn('Polyfill initialization failed:', error);
+    // Never throw from polyfill init
+    if (import.meta.env.DEV) {
+      // eslint-disable-next-line no-console
+      console.warn('[polyfills] initialization warning:', error);
+    }
   }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)